migrations/admin_maps_test.go

```diff
@@ -0,0 +1,129 @@
+package migrations
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	evgdb "github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/mock"
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type adminMapSuite struct {
+	env      *mock.Environment
+	session  db.Session
+	database string
+	now      time.Time
+	cancel   func()
+	suite.Suite
```

Comment:
I added migrationSuite, which does all the anser set up for you. Swap out your suite.Suite for migrationSuite, and you can drop all the anser specific stuff from this test